### ISSUE TITLE
[1.15.2] Crash when start MC

### ISSUE THREAD

**Issue by zanecek** [Vazkii/Patchouli#239]
[crash-2020-05-16_01.46.30-client.txt](https://github.com/Vazkii/Patchouli/files/4637510/crash-2020-05-16_01.46.30-client.txt)

**Comment by zanecek** [comment]
With older FORGE version it is OK...

**Comment by Geethebluesky** [comment]
Same crash on my end: [logfile](https://pastebin.com/UpS2AjnG)

**Comment by Devintrainer** [comment]
I don't like you now, Patchouli!

**Comment by ailailou** [comment]
Same issue here, exact same kind of crash log lines. Latest version of forge + optiforge.

**Comment by Hubry** [comment]
Please post the `debug.log` or `latest.log`, that crash log exception usually hides the actual one that happened earlier.

**Comment by ailailou** [comment]
I wished I saved the log file, and I'm used to this on my daily tester job :(.

After my comment I did a clean Minecraft  install including removing all my 91 mods and starting from scratch.

For some background, I decided to update optifine to latest prerelease 18, which required an update on latest forge 31.2.0. This triggered the patchouli crash (before this I was on forge 31.1.86 and earlier optifine preview).

In any case I started from scratch and I ran the following setting:
- forge-1.15.2-31.2.0
- preview_OptiFine_1.15.2_HD_U_G1_pre18
- MixinBootstrap-1.0.3
- OptiForge-MC1.15.2-0.1.19
- Patchouli-1.15.2-1.2-33

With only these 5 mods game does not crash any longer, so it must have been some other mod from my pack. I need to go over all the mods in there and see if any need updating, this will take a few days. I will try and see if I can reproduce the issue afterwards.

Another thing I can add is that before deciding to start from scratch I removed patchouli mod only, and afterwards I had a biome o plenty crash so yeah.
